cmd/vegawallet/commands/key_isolate.go

```diff
@@ -102,16 +102,22 @@ func BuildCmdIsolateKey(w io.Writer, handler IsolateKeyHandler, rf *RootFlags) *
 		"",
 		"Path to the file containing the wallet's passphrase",
 	)
+	cmd.Flags().StringVarP(&f.IsolatedWalletPassphraseFile,
+		"isolated-passphrase-file", "i",
```

Comment:
I think this flag should be named `isolated-wallet-passphrase-file`, otherwise it conveys another meaning.